xml/remote_administration.xml

```diff
@@ -298,42 +296,45 @@ nodeattr [-f genders] -k
     Check permissions, owner and file type of the key file located under
     <filename>/etc/munge/munge.key</filename> on the local system:
    </para>
-   <screen># stat --format "%F %a %G %U %n" /etc/munge/munge.key</screen>
+<screen># stat --format "%F %a %G %U %n" /etc/munge/munge.key</screen>
    <para>The settings should be:</para>
-   <screen>400 regular file munge munge /etc/munge/munge.key</screen>
+<screen>400 regular file munge munge /etc/munge/munge.key</screen>
    <para>Get md5sum of munge.key:</para>
-   <screen># md5sum /etc/munge/munge.key</screen>
+<screen># md5sum /etc/munge/munge.key</screen>
    <para>Copy key to list of nodes:</para>
-   <screen># pdcp -R ssh -w &lt;hostlist&gt; /etc/munge/munge.key
+<screen># pdcp -R ssh -w &lt;hostlist&gt; /etc/munge/munge.key
    /etc/munge/munge.key </screen>
    <para>Check key settings in remote host:</para>
-   <screen>pdsh -R ssh -w &lt;hostlist&gt; stat --format \"%F %a %G %U %n\"
-   /etc/munge/munge.key
-   pdsh -R ssh -w &lt;hostlist&gt; md5sum /etc/munge/munge.key
-   </screen>
-   <para>make sure they match.</para>
+<screen>pdsh -R ssh -w &lt;hostlist&gt; stat --format \"%F %a %G %U %n\"
+/etc/munge/munge.key
+pdsh -R ssh -w &lt;hostlist&gt; md5sum /etc/munge/munge.key</screen>
+   <para>Make sure that they match.</para>
   </sect2>
   <sect2>
-   <title>Enabling and starting MUNGE</title>
+   <title>Enabling and starting Munge</title>
```

Comment:
MUNGE is the correct spelling (see the homepage of the project that I linked this morning):
```suggestion
   <title>Enabling and starting MUNGE</title>
```